src/lambda/local/debugConfiguration.ts

```diff
@@ -23,13 +23,13 @@ export interface NodejsDebugConfiguration extends DebugConfiguration {
     readonly skipFiles?: string[]
 }
 
+export interface DebugConfigurationPathMapping {
```

Comment:
Each debugger has its own structure for path mappings. For example, for `coreclr` (.NET Core) it's

```typescript
{
    [remotePath: string]: /* local path */ string
}
```

Suggest to rename to something like `PythonDebugConfigurationPathMapping` or `PythonPathMapping` to clarify that it is debugger-specific.

Reply:
great call -- I didn't realize other runtimes had different path mapping structures